Unit tests fail in CI on Windows

They fail like this, probably due to newer Chromium using different PNG encoding:
```
FAILED tests/unit/bokeh/io/test_export.py::test_get_svg_no_svg_present[chromium] - assert ['<svg versio...mCC"/></svg>'] == ['<svg versio...mCC"/></svg>']
  At index 0 diff: '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" height="20"><defs/><image width="20" height="20" preserveAspectRatio="none" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAFNJREFUOE9jZKAyYKSyeQyjBkJC9D8DgwMDBBMCBxgZGA4gK8Iahv8ZGBqA5tYTMo2BgaERaABILRyMGgiPlNEwxJmABi7ZUDenEJFDcCoZgcUXALJrIRUwmbFzAAAAAElFTkSuQmCC"/></svg>' != '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" height="20"><defs/><image width="20" height="20" preserveAspectRatio="none" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAUUlEQVQ4T2NkoDJgpLJ5DKMGQkL0PwODA5ACYULgADDMDiArwhqGQAMbgIrqCZkGlG8EGgBSCwejBsIjZTQMcSaggUs21M0pROQQnEpGYPEFALJrIRV3ULevAAAAAElFTkSuQmCC"/></svg>'
  Full diff:
    [
     '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" '
     'height="20"><defs/><image width="20" height="20" preserveAspectRatio="none" '
  -  'href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAUUlEQVQ4T2NkoDJgpLJ5DKMGQkL0PwODA5ACYULgADDMDiArwhqGQAMbgIrqCZkGlG8EGgBSCwejBsIjZTQMcSaggUs21M0pROQQnEpGYPEFALJrIRV3ULevAAAAAElFTkSuQmCC"/></svg>',
  +  'href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAFNJREFUOE9jZKAyYKSyeQyjBkJC9D8DgwMDBBMCBxgZGA4gK8Iahv8ZGBqA5tYTMo2BgaERaABILRyMGgiPlNEwxJmABi7ZUDenEJFDcCoZgcUXALJrIRUwmbFzAAAAAElFTkSuQmCC"/></svg>',
    ]
```

--- a/tests/unit/bokeh/io/test_export.py
+++ b/tests/unit/bokeh/io/test_export.py
@@ -17,6 +17,7 @@ import pytest ; pytest
 #-----------------------------------------------------------------------------
 
 # Standard library imports
+import base64
 import re
 import sys
 from typing import TYPE_CHECKING
@@ -195,30 +196,21 @@ def test_get_svg_no_svg_present(webdriver: WebDriver) -> None:
     with silenced(MISSING_RENDERERS):
         svgs = bie.get_svg(layout, driver=webdriver)
 
-    def output(data: str) -> list[str]:
-        return [
-            '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" height="20">'
-                '<defs/>'
-                f'<image width="20" height="20" preserveAspectRatio="none" href="data:image/png;base64,{data}"/>'
-            '</svg>',
-        ]
+    assert isinstance(svgs, list) and len(svgs) == 1
+    [svg] = svgs
 
-    chrome_data = (
-        "iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAAXNSR0IArs4c6QAAAFNJ"
-        "REFUOE9jZKAyYKSyeQyjBkJC9D8DgwMDBBMCBxgZGA4gK8Iahv8ZGBoYGBjqCZnGwMDQ"
-        "yAhRCwejBsIjZTQMcSaggUs21M0pROQQnEpGYPEFALJrIRXAq4rZAAAAAElFTkSuQmCC"
-
-    )
-    firefox_data = (
-        "iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAUUlEQVQ4T2NkoDJgpLJ5D"
-        "KMGQkL0PwODA5ACYULgADDMDiArwhqGQAMbgIrqCZkGlG8EGgBSCwejBsIjZTQMcSaggUs"
-        "21M0pROQQnEpGYPEFALJrIRV3ULevAAAAAElFTkSuQmCC"
+    pattern = re.compile(
+        '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="20" height="20">'
+            '<defs/>'
+            '<image width="20" height="20" preserveAspectRatio="none" href="data:image/png;base64,([^"]*)"/>'
+        '</svg>',
     )
 
-    if webdriver.name == "chrome":
-        assert svgs == output(chrome_data)
-    else:
-        assert svgs == output(firefox_data)
+    result = pattern.match(svg)
+    assert result is not None
+
+    (data,) = result.groups()
+    assert base64.b64decode(data).startswith(b"\x89PNG\r\n")
 
 @pytest.mark.selenium
 def test_get_svg_with_svg_present(webdriver: WebDriver) -> None:

